### ISSUE TITLE
PS version support and it nearly being there on CS6 (a fix for which would be great, if anyone has any ideas)

### ISSUE THREAD

**Issue by jon709** [webmproject/WebPShop#65]
I notice that the main page says both (my bold):

"Photoshop 23.2 and newer natively supports WebP. However some features such as preview at encoding and animations are missing. WebPShop can still be installed and used for these use cases **and/or for Photoshop 23.1 and below.**"
(Which would include back to CS6 you might, or might not, think.)

"Only the latest Photoshop release is supported."
(Which would be 25.6 BTW.)

Anyway I decided to try 0-4-3 in CS6 x64 Extended under Win10 Pro x64. Once you put it in:
C:\Program Files\Common Files\Adobe\Plug-Ins\CS6
(Rather than ...\CC.) It's pretty good. Loading and saving both seem to work fine (many thanks for that, saves me a lot of effort), only the preview doesn't work. If there's any simple way to fix the preview that would be great!

P.S. I even read the small print and did a CLA...

**Comment by y-guyon** [comment]
Thank you for your interest in WebPShop and for sharing this.

> If there's any simple way to fix the preview that would be great!

"Only the latest Photoshop release is supported" because I do not have access to other versions of Photoshop and because I do not have the time to handle them anyway. Feel free to investigate and suggest a fix though.

**Comment by tysonmatanich** [comment]
I agree that the README needs updated as it contains the following conflicting statements:

- "WebPShop can still be installed and used for these use cases and/or for Photoshop 23.1 and below."
- "Only the latest Photoshop release is supported."

It would be nice to include what older version may work and which ones don't work. If no CS versions will work just state that so people don't waste their time. Perhaps only CC versions or CC versions starting at a specific version.

**Comment by y-guyon** [comment]
> I agree that the README needs updated

Done. Is the current wording more accurate?

> It would be nice to include what older version may work and which ones don't work.

I agree but I do not have the time nor the setup for that. Feel free to send a pull request though.

**Comment by tysonmatanich** [comment]
I think this wording is a lot clearer and sets the proper expectations.
